pkg/util/log/strip.go

```diff
@@ -57,7 +56,12 @@ func init() {
 		Hints: []string{"community_string", "authKey", "privKey"},
 		Repl:  []byte(`$1 ********`),
 	}
-	replacers = []Replacer{apiKeyReplacer, appKeyReplacer, uriPasswordReplacer, passwordReplacer, tokenReplacer, snmpReplacer}
+	certReplacer = Replacer{
```

Comment:
So my biggest concern here is we'll be running the regex, on every line logged (as called by the `scrub` function in the same package), this could drive CPU usage up quite a bit in a `debug` scenario. It would be nice to try to use a `Hint` to help keep this side-effect under control.

Reply:
I jumped straight into the code not having have seen the previous review. I see we're trying to keep track of the performance impact 👌

Reply:
You're right about the `Hint`: I should add `BEGIN` (or `END`) as a hint so that it doesn't try to apply the real regexp if it isn't here. I'm going to try it and see if that improves the benchmarks (which should theoretically happen, since we don't put certificates in our benchs).
Note: should we add benchmarks where there can be certificates in the logs then?

Reply:
I also thought of there's another way to do this that might be less costly, but it's more convoluted and requires more work, so I don't know if that's worth the time.
Basically it would require creating a new type of replacer with a `StartRegex` and an `EndRegex`. We feed it the lines one by one: 
- if the line doesn't match the `StartRegex`, we write the line to the result
- if the line matches the `StartRegex`, we start keeping the lines (without writing the result) until we match the `EndRegex`. At this point, we replace everything that was between these two lines by whatever was set in the `Repl` or `ReplFunc` fields.
Moreover, we could add a `StartHint` and an `EndHint` for these two regexes (since we're matching by line).